src/interfaces/swig/SGBase.i

```diff
@@ -446,6 +469,31 @@ namespace shogun
     }
 }
 
+%pythoncode %{
+	def _internal_get_param(self, name):
```

Comment:
Yes, something similar might be possible with other interpreted languages right?

Reply:
definitely, but let's first start with python to see how it goes
Such a nice API simplification.
BTW once this is merged, you can change the [json](https://github.com/shogun-toolbox/shogun/blob/develop/examples/meta/generator/targets/cpp.json#L97) that does the `get_real_vector` names and unify it to a simple get. Then we have only moved the place where "real" "real_vector", etc are explicitly stated, rather than added this at another place in the codebase